src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -142,6 +140,14 @@ public void markAssignment(Name name, Assignment key) {
         persons.personWithSameName(name).getAssignments().done(key);
     }
 
+    /**
+     * Retrieve the assignment list of the identified person {@code name} from this {@code AddressBook's person list}.
+     * Person with {@code name} must exist in the person list.
+     */
+    public ObservableList<Assignment> getPersonAssignmentList(Name name) {
+        return persons.assignmentsOfPersonWithSameName(name).asUnmodifiableObservableList();
```

Comment:
Perhaps you can consider changing the name of ```assignmentOfPersonWithSameName``` to a verb/action?

Reply:
Is it better to be getAssignmentOfPersonWithSameName or is getPersonAssignmentList is enough. There is another similar method in the class call personWithSameName that was already inside I think, should I change that to a verb for consistency as well?